go_dependencies.bzl

```diff
@@ -1569,6 +1569,7 @@ def go_dependencies():
         name = "org_golang_x_sys",
         importpath = "golang.org/x/sys",
         sum = "h1:SqMFp9UcQJZa+pmYuAKjd9xq1f0j5rLcDIk0mj4qAsA=",
+        patches = ["//:patches/org_golang_x_sys/unix_add_Fsconfig_syscall_on_linux.patch"],
```

Comment:
Nits:
- Please name the diff file .diff instead of .patch. That's also how the other files in that directory are called.
- I think your PR doesn't actually include this file.
- Could you please put the number of the upstream PR in the name of the diff file? That way it's easier to find going forward.

Have you also forked the upstream PR as discussed?

Reply:
1:  done
2:  done
3:  done

4: No, but I found that IlyaHanov started the process: https://github.com/golang/go/issues/59537